lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1053,6 +1077,18 @@ def test_access(self, user, status):
             memberships = self.get_membership_list(200, {'team_id': self.solar_team.team_id})
             self.assertEqual(memberships['count'], 2)
 
+            add_method = 'joined_from_team_view' if user == 'student_enrolled_not_on_team' else 'added_by_another_user'
```

Comment:
Why do we expect `'added_by_another_user'` for staff, course_staff, and community_ta? Aren't they adding themselves?